web/layout.go

```diff
@@ -37,7 +37,7 @@ type SubmenuItem struct {
 }
 
 var defaultLayoutData = LayoutData{
-	Title:     "Trento Console",
+	Title:     "Trento Console for SAP Applications",
```

Comment:
I would leave it as `Trento console`, but I leave this decision to @lee-martin 
Otherwise, we will need to widen back again the sidebar